conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -151,6 +151,11 @@ public JavaFile generateServiceHandler(
                 .addField(FieldSpec.builder(serviceClass, DELEGATE_VAR_NAME, Modifier.PRIVATE, Modifier.FINAL)
                         .build())
                 .addMethod(MethodSpec.methodBuilder("of")
+                        .addAnnotation(Deprecated.class)
+                        .addJavadoc(
+                                "@Deprecated: You can now use {@link $1T} directly as it implements {@link $2T}.",
+                                serviceClass,
+                                UndertowService.class)
```

Comment:
I don't think we should deprecate this if our recommendation for the diamond problem is to call it directly.

Reply:
Good point, removed the deprecation flag.